src/libraries/sendtohelix.proj

```diff
@@ -62,7 +67,7 @@
     <Message Condition="'$(TestArchiveRuntimeFile)' != ''" Importance="High" Text="TestArchiveRuntimeFile: $(TestArchiveRuntimeFile)" />
 
     <!-- Re-invoke MSBuild on this project to create the correlation payload -->
-    <MSBuild Projects="$(MSBuildProjectFile)" Targets="PrepareCorrelationPayloadDirectory" Properties="Scenarios=$(_Scenarios)" />
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="PrepareCorrelationPayloadDirectory" Properties="$(_PropertiesToPass)" />
```

Comment:
This ends up passing *a lot* more global properties than were previously passed.  Why do we need to do that?  Moreover, why are we even doing an MSBuild invocation rather than just running the target in the same evaluation?

Reply:
Is it a problem to pass through more properties?

I need it to pass through the `SuperPmiCollect` property, at least. It seemed simpler and more consistent to just pass through all the ones we pass through on the per-scenario msbuild re-invocation. It's a bit unfortunate that the libraries and runtime versions of this file are (so) different: it (src\tests\Common\helixpublishwitharcade.proj) passes through all the properties.

I can't answer the question about re-invocation. Maybe it provides easier-to-control ordering? The coreclr .proj file does this too (and for more cases, it appears).